src/sql/magic_plot.py

```diff
@@ -62,7 +62,7 @@ def execute(self, line="", cell="", local_ns=None):
             column = cmd.args.column
 
         if not cmd.args.line:
-            raise ValueError(
+            raise exceptions.PloomberValueError(
```

Comment:
We shouldn't change the error type

Reply:
Fix in 
[Fix: use modify_exceptions decorator](https://github.com/ploomber/jupysql/pull/106/commits/8c9ce8c8b04c56916bab07e3bc05bc6798d477c3)